[2023-07-28T03:30:13.020+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-27T00:00:00+00:00 [queued]>
[2023-07-28T03:30:13.030+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-27T00:00:00+00:00 [queued]>
[2023-07-28T03:30:13.030+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-28T03:30:13.031+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-28T03:30:13.031+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-28T03:30:13.046+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-27 00:00:00+00:00
[2023-07-28T03:30:13.049+0000] {standard_task_runner.py:55} INFO - Started process 2749 to run task
[2023-07-28T03:30:13.052+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-27T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp7k9b_n83']
[2023-07-28T03:30:13.052+0000] {standard_task_runner.py:83} INFO - Job 3: Subtask create_table
[2023-07-28T03:30:13.103+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-27T00:00:00+00:00 [running]> on host b81352b01223
[2023-07-28T03:30:13.157+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-27T00:00:00+00:00
[2023-07-28T03:30:13.165+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-28T03:30:13.167+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users
            (
                lastname TEXT NOT NULL,
                firstname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            )
        , parameters: None
[2023-07-28T03:30:13.168+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-28T03:30:13.176+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230727T000000, start_date=20230728T033013, end_date=20230728T033013
[2023-07-28T03:30:13.223+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-28T03:30:13.238+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-28T09:03:37.549+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-27T00:00:00+00:00 [queued]>
[2023-07-28T09:03:37.557+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-27T00:00:00+00:00 [queued]>
[2023-07-28T09:03:37.557+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-28T09:03:37.557+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-28T09:03:37.558+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-28T09:03:37.569+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-27 00:00:00+00:00
[2023-07-28T09:03:37.573+0000] {standard_task_runner.py:55} INFO - Started process 9216 to run task
[2023-07-28T09:03:37.575+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-27T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpi2t39b5n']
[2023-07-28T09:03:37.577+0000] {standard_task_runner.py:83} INFO - Job 8: Subtask create_table
[2023-07-28T09:03:37.625+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-27T00:00:00+00:00 [running]> on host b81352b01223
[2023-07-28T09:03:37.676+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-27T00:00:00+00:00
[2023-07-28T09:03:37.684+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-28T09:03:37.687+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users
            (
                lastname TEXT NOT NULL,
                firstname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            )
        , parameters: None
[2023-07-28T09:03:37.687+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-28T09:03:37.693+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230727T000000, start_date=20230728T090337, end_date=20230728T090337
[2023-07-28T09:03:37.707+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-28T09:03:37.731+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
